Dockerfile

```diff
@@ -1,13 +1,12 @@
 FROM node:8
 
-RUN mkdir -p /usr/src
 WORKDIR /usr/src/
 
-COPY ./ .
+ADD . /usr/src
+
 RUN chown -R node:node .
 
 USER node
 
-RUN npm install
-RUN ./node_modules/.bin/lerna link
-RUN ./node_modules/.bin/lerna bootstrap --no-ci --loglevel verbose
+RUN npm config set unsafe-perm true
```

Comment:
i wasn't' super keen to use this config options but couldn't get it building without...thoughts appreciated.

Reply:
We use the same config to build the JS for zooniverse.org.
https://github.com/zooniverse/Panoptes-Front-End/blob/fb066b66f67f02c759b42b4ff95dba0564c56c39/Dockerfile#L13

Set it to false if npm is run as root, but that sounds less safe to me.
https://docs.npmjs.com/misc/config#unsafe-perm